parm/land/letkfoi/letkfoi.yaml

```diff
@@ -16,7 +16,7 @@ geometry:
       filetype: fms restart
       skip coupler file: true
       state variables: [orog_filt]
-      datapath: !ENV ${FIXgfs}/fix_orog/${CASE}/
+      datapath: !ENV ${FIXgfs}/orog/${CASE}/
```

Comment:
@jiaruidong2017 do I need to change the path of files anywhere for testing purposes?

Reply:
@CoryMartin-NOAA The yaml file for ctests was provided in GDASApp. Therefore, there is no need to make other changes. Thanks.